### PR TITLE
{lang} [intel/2016b] PyCUDA/2016.1.2 (REVIEW)

### PR DESCRIPTION
**Pull request by smoors**
This version does not work:
PyCUDA-2016.1.2-intel-2017a-Python-2.7.13.eb

### CODE REVIEW

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
nic66 - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 2.7.5
See https://gist.github.com/6cea524e38a404d30cf12a105304a7d8 for a full test report.

File: easybuild/easyconfigs/p/PyCUDA/PyCUDA-2016.1.2-intel-2016b-Python-2.7.12.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+version = '2016.1.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://mathema.tician.de/software/pycuda'
+description = """PyCUDA lets you access Nvidia’s CUDA parallel computation API from Python."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
```
Comment (boegel) [inline easybuild/easyconfigs/p/PyCUDA/PyCUDA-2016.1.2-intel-2016b-Python-2.7.12.eb]: @wpoely86 please keep an eye on the presence of checksums, we should start forcing that in Travis soon (cfr. #5005)

**Comment by boegel** [comment]
@smoors: for future reference, how does the existing `PyCUDA-2016.1.2-intel-2017a-Python-2.7.13.eb` not work?

**Comment by smoors** [comment]
CUDA/8.0.61 needs a GCC version <= 5, but intel/2017a uses GCC 6

**Comment by boegel** [comment]
@smoors Do you mind copy-pasting the error message here, so people can track it down if they run into it.

Does it make sense to retract the previous easyconfig, since it just doesn't work?
Can we add a sanity check command that avoids this from happening again?

Also, please submit a follow-up PR that adds a SHA256 checksum for this easyconfig.

**Comment by wpoely86** [comment]
This is just general CUDA stuff. It doesn't work with recent GCC, it will complain loudly and the error is already all over the internet.

It would be good to retrack the other easyconfig as it can never work with the current versions of CUDA.

**Pull request merged**